syntax = "proto3";

package protos;
option go_package = "orch-go/gen/protos;protos";
option csharp_namespace = "Core";

import "custom_types.proto";
import "google/protobuf/empty.proto";
import "fee_type.proto";

service FeeTypeAnalyticService {
  rpc ProcessGetAll (GetAllRequest) returns (GetAllFeeTypesResponse);
  rpc ProcessGetById (GetFeeTypeByIdRequest) returns (FeeTypeModel);
  rpc ProcessGetByIds (GetFeeTypeByIdsRequest) returns (GetAllFeeTypesResponse);
  rpc ProcessGetCount (google.protobuf.Empty) returns (CountResponse);
}
